clinvar_ingest/api/model/requests.py

```diff
@@ -108,14 +109,59 @@ def _validate(cls, v):  # pylint: disable=E0213
         return v
 
 
+class GcsBucketName(RootModel):
+    root: str
+
+    @validator("root")
+    def _validate(cls, v):  # pylint: disable=E0213
+        alpha_lower = "abcdefghijklmnopqrstuvwxyz"
```

Comment:
```
import string
string.ascii_lowercase
```
would maybe be useful here?

Reply:
(+ the other attributes of that module, like `ascii_letters` and `digits`)

Reply:
i was wondering why this def doesn't use a regex. Is the approach in this commit a better performer or easier to maintain?

Reply:
yeah.... I'm not sure... I'd tend to prefer something explicit over a regex, just because those tend to be harder to maintain.

I think we should be precise though... just in reading [this doc](https://cloud.google.com/storage/docs/buckets) it looks like we're not validating the spec correctly.  

Could we just do something simple like checking if the string starts with `gs://`?

Reply:
I'm just going to delete this. I did something similar for `bigquery_full_table_id` but using regex, which is probably nicer. 

For this one I decided not to use it because google will validate the input bucket name anyways.

Reply:
agree with the decision!